.github/scripts/utils.py

```diff
@@ -0,0 +1,29 @@
+from typing import Optional, Dict, Any, List
```

Comment:
nit: since this only has rockset commands, we could call the file a slightly less generic "rockset_utils"